addons/binding/org.openhab.binding.amazonechocontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,425 @@
+            <channel id="subtitle2" typeId="subtitle2" />
+            <channel id="providerDisplayName" typeId="providerDisplayName" />
+            <channel id="bluetoothId" typeId="bluetoothId" />
+            <channel id="bluetoothIdSelection" typeId="bluetoothIdSelection" />
+            <channel id="bluetooth" typeId="bluetooth" />
+            <channel id="bluetoothDeviceName" typeId="bluetoothDeviceName" />
+            <channel id="radioStationId" typeId="radioStationId" />
+            <channel id="radio" typeId="radio" />
+            <channel id="amazonMusicTrackId" typeId="amazonMusicTrackId" />
+            <channel id="amazonMusicPlayListId" typeId="amazonMusicPlayListId" />
+            <channel id="amazonMusicPlayListIdLastUsed" typeId="amazonMusicPlayListIdLastUsed" />
+            <channel id="amazonMusic" typeId="amazonMusic" />
+            <channel id="remind" typeId="remind" />
+            <channel id="playAlarmSound" typeId="playAlarmSound" />
+            <channel id="playFlashBriefing" typeId="playFlashBriefing" />
+            <channel id="playTrafficNews" typeId="playTrafficNews" />
+            <channel id="playWeatherReport" typeId="playWeatherReport" />
+            <channel id="playGoodMorning" typeId="playGoodMorning" />
+            <channel id="startRoutine" typeId="startRoutine" />
+            <channel id="playMusicProvider" typeId="playMusicProvider" />
+            <channel id="playMusicVoiceCommand" typeId="playMusicVoiceCommand" />
+        </channels>
+        <representation-property>serialNumber</representation-property>
+        <config-description>
+            <parameter name="serialNumber" type="text" required="true">
+                <label>Serial Number</label>
+                <description>You will find the serial number of your device in the Alexa app</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+    <thing-type id="flashbriefingprofile">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="account" />
+        </supported-bridge-type-refs>
+        <label>Flash Briefing Profile</label>
+        <description>Store and load a flash briefing configuration</description>
+        <channels>
+            <channel id="save" typeId="save" />
+            <channel id="active" typeId="active" />
+            <channel id="playOnDevice" typeId="playOnDevice" />
+        </channels>
+    </thing-type>
+    <channel-type id="save">
+        <item-type>Switch</item-type>
+        <label>Save</label>
+        <description>Save the current flash briefing configuration (Write only)</description>
+    </channel-type>
+    <channel-type id="active">
+        <item-type>Switch</item-type>
+        <label>Active</label>
+        <description>Activate this flash briefing configuration</description>
+    </channel-type>
+    <channel-type id="playOnDevice">
+        <item-type>String</item-type>

```

Comment:
I can not comment your comment above. So I do it here: the flashbriefingprofile encapsulate an  arrangement of news channels, so this is not an real hardware device. The user can create a flashbriefingprofile for each needed arrangement. So its an 1:n relation between account and flahsbriefingprofile

To your next comment: It's not a technical configuration parameter. There exist a stateprovider for it, and the user can select the device in an dropdown in the paper UI. The "playFlashBriefing" channel only starts the current active flash briefing configuration. But I would be possible to do it in the opposite way, so that the user have a drop down in the echo thing, but I think this makes no really difference.